perl/bin/battenberg.pl

```diff
@@ -171,6 +195,24 @@ sub setup {
 
 	pod2usage(-msg  => "\nERROR: Options must be defined.\n", -verbose => 2,  -output => \*STDERR) unless($defined);
 
+  if(!defined($opts{'noclean'})){
+    $opts{'noclean'} = 0;
+  }
+
+  #Check that outdir exists and is writable. Create directory if it does not exist
+  if (defined $opts{'outdir'}) {
+    make_path($opts{'outdir'}) unless(-d $opts{'outdir'});
+    unless (-w $opts{'outdir'}) {
```

Comment:
`make_path` will die on fatal errors (such as being unable to create the path) so there is no need to check if the area is writable anymore.  The error is pretty sensible too:

```perl
perl -e 'use File::Path qw(make_path); use autodie qw(:all); make_path(q{/giggidy});'
```
Gives
```
mkdir /giggidy: Permission denied at -e line 1.
```